### PR TITLE
Stat tracker spec test

### PR DESCRIPTION
**Pull request by priyapower**
Author: Logan/Priya

Updates StatTracker for newest code
FOUND BUG AND FIXED: Season --> best and worst coach

### CODE REVIEW

**Comment by lkriffell** [comment]
reviewed by: Logan Riffell

File: lib/season_statistics.rb [hunk 1]
```diff
@@ -11,76 +11,56 @@ def initialize
...
-    winningest_coach = @coach_by_team_id[max_team_id]
+    @win_percentage_by_coach.invert.max[1]
   end
 
   def worst_coach(season)
     winningest_and_worst_suite(season)
-    min_team_id = @win_percentage.invert.min[1]
-    worst_coach = @coach_by_team_id[min_team_id]
+    @win_percentage_by_coach.invert.min[1]
   end
```
Comment (lkriffell) [inline lib/season_statistics.rb:65]: Nice refactoring! Lots of unneeded code removed

File: test/game_statistics_test.rb [hunk 2]
```diff
@@ -19,13 +19,30 @@ def test_game_statistics_can_inherit_game_array
...
+  def test_percentage_home_wins
     assert_equal 0.44, @game_statistics.percentage_home_wins
+  end
+
+  def test_percentage_vistor_wins
     assert_equal 0.36, @game_statistics.percentage_visitor_wins
+  end
+
+  def test_percentage_ties
     assert_equal 0.20, @game_statistics.percentage_ties
```
Comment (lkriffell) [inline test/game_statistics_test.rb:46]: I see you imported all the new code

File: lib/season_statistics.rb (hunk 1 above)
Comment (priyapower) [reply lib/season_statistics.rb:65]: WOOO!

File: test/game_statistics_test.rb (hunk 2 above)
Comment (priyapower) [reply test/game_statistics_test.rb:46]: I'm glad this still runs in spec harness. I was afraid

**Pull request merged**